data/mods/Safe_autodoc/modinfo.json

```diff
@@ -1,13 +0,0 @@
-[
-  {
-    "type": "MOD_INFO",
-    "ident": "safeautodoc",
```

Comment:
Please add this to the mod migration list.
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/mods/replacements.json
Just
```
[
  "safeautodoc"
]
```